kfp/kfp_support_lib/kfp_v1_workflow_support/pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [project]
 name = "data_prep_toolkit_kfp_v1"
 version = "0.2.1.dev3"
-requires-python = ">=3.10,<3.12"
+requires-python = ">=3.10"
```

Comment:
Again, can we use values from make.default

Reply:
it is, now